test/helpers.js

```diff
@@ -31,20 +31,30 @@ exports.compile = function compile(compiler) {
   });
 };
 
-exports.createCompiler = function createCompiler(options) {
-  const compiler = webpack(options || {
-    bail: true,
+exports.createCompiler = function createCompiler(options = {}) {
+  const compiler = webpack({
     cache: false,
     entry: `${__dirname}/stubs/entry.js`,
+    resolve: {
+      unsafeCache: false,
```

Comment:
If it is helpful for debugging and less likely to introduce conflicts it's ok, but if it doesn't affect unit tests better reduce the noise where possible 🐤